examples/ui/text_wrap.rs

```diff
@@ -0,0 +1,67 @@
+use bevy::prelude::*;
+
+fn main() {
+    App::new()
+        .insert_resource(ClearColor(Color::rgb(0.4, 0.4, 0.4)))
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup)
+        .run();
+}
```

Comment:
I found two bugs in the example:

1. When I resize the window (which itself resizes the `NodeBundle` containing the wrapping text), the text wrap doesn't update properly. (I think to fix this you should look at lines 80 to 95 in `widget/text.rs`)
2. For some reasons, the text that says "It should only fill half the container size" only fills **a quarter** the container size, the one that says "It should fill the whole container" only fills half of it. (no idea why that though)

Reply:
for 2. by container I mean the item that wraps the text element, the container itself only fills half the window, so the text content should wrap at half of its container, making the text only a quarter of the window size and half of the container.

Reply:
This should now resize itself when re-scaling the window

Reply:
The example for me is different from the screenshot you posted in the initial PR comment, I was referring to the container (the darker area of the screen) as well, it looks like this to me.

![text_wrap_pr_bug-2022-08-01](https://user-images.githubusercontent.com/26321040/182147266-a91a62ab-bd7c-420a-a995-b6b92774297d.png)

I'm really not sure what's causing this, my best guess would be something to do with the `scale_factor`.

The code otherwise looks good to merge though.

Reply:
Oh right, sorry I misunderstood you. That's really strange, yeah I'll have to investigate

Reply:
Some borders around the text-nodes would really help here, but I think we have implemented that just yet so is something we could improve later